api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -21,4 +21,73 @@
     <include file="liquibase-update-to-2.2.xml"/>
     <include file="liquibase-update-to-2.3.xml"/>
 
+
+<changeSet id="TRUNK-3343-assigning-new-app-privileges" author="mozzy">
+		
```

Comment:
Shouldn't you add some preconditions?

Reply:
oh ya but in this case i failed to figure out an appropriate pre condition. May be u can give me a hint

Reply:
i added that pre-condition